front/components/items/Appointment/helper.ts

```diff
@@ -28,3 +28,22 @@ export const getDateOfAppointments = ({ appointments }: props) => {
         }
     });
 };
+
+export const getIdOfAppointment = ({
```

Comment:
missleading name. This function is not returning id, its returning whole appointment, and also its returning it by some conditions. Think about more accurate name that will say that you are returning appointment that meets conditions x,y,z